cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfJointComponent.cpp

```diff
@@ -0,0 +1,78 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfJointComponent.h"
+
+#include <iostream>
+
+#include <PhysicsEngine/PhysicsConstraintComponent.h>
+
+#include "CoreUtils/Assert.h"
+#include "UrdfLinkComponent.h"
+#include "UrdfParser.h"
+
+void UUrdfJointComponent::initialize(UrdfJointDesc* joint_desc, UUrdfLinkComponent* parent_link, UUrdfLinkComponent* child_link)
+{
+    this->joint_type_ = joint_desc->type_;
+
+    ConstraintInstance.SetDisableCollision(true);
+
+    ConstraintInstance.ProfileInstance.ConeLimit.Swing1Motion = EAngularConstraintMotion::ACM_Locked;
+    ConstraintInstance.ProfileInstance.ConeLimit.Swing2Motion = EAngularConstraintMotion::ACM_Locked;
+    ConstraintInstance.ProfileInstance.TwistLimit.TwistMotion = EAngularConstraintMotion::ACM_Locked;
+    ConstraintInstance.UpdateAngularLimit();
+
+    ConstraintInstance.ProfileInstance.AngularDrive.AngularDriveMode = EAngularDriveMode::TwistAndSwing;
+    ConstraintInstance.ProfileInstance.ConeLimit.bSoftConstraint = true;
+    ConstraintInstance.ProfileInstance.TwistLimit.bSoftConstraint = false;
+
+    const float M_TO_CM = 100.0f;
```

Comment:
Move this constant to the top of the file, or if it is a local variable, just name it `float m_to_cm`. By convention, we don't use `const` for local variables because it's too much work to do it consistently.

Reply:
fixed